package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "bakeryjs",
-	"version": "0.0.3",
+	"version": "0.0.4",
```

Comment:
Please don't do version bumps in PRs, it messes up `npm version` command.

Reply:
Reverted.